tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -7550,20 +7558,29 @@ node_modules/chrome-devtools-frontend/front_end/mobile_throttling/ThrottlingMana
 node_modules/chrome-devtools-frontend/front_end/mobile_throttling/ThrottlingManager.js(266,15): error TS2339: Property 'singleton' does not exist on type 'Window'.
 node_modules/chrome-devtools-frontend/front_end/mobile_throttling/ThrottlingPresets.js(14,2): error TS1131: Property or signature expected.
 node_modules/chrome-devtools-frontend/front_end/mobile_throttling/ThrottlingPresets.js(20,18): error TS2300: Duplicate identifier 'Conditions'.
+node_modules/chrome-devtools-frontend/front_end/mobile_throttling/ThrottlingPresets.js(22,30): error TS2694: Namespace 'MobileThrottling' has no exported member 'Conditions'.
```

Comment:
These new errors arise from the unsupported form of typedef:

```js
/** @typedef {{ ... }} */
Entity.Name.Reference;
```

These have never been supported, so I suspect that  #24867, which now checks type tags, is just now reporting these errors. That is, I believe that the places the errors appear are unused in the code, so would not have had an error without the explicit check that I added.

Reply:
We _do_ support that style of typedef though, afaik. I've seen the codepaths for it. The intent is definitely there, anyway.

Reply:
Only for simple identifiers: `/** @typedef {*} */ var A`, not `/** @typedef {*} */ var A.B.C;`

Previously the extensions that supported both entity names and non-comment-names were so (1) hacky and (2) orthogonal that it wasn't worth the effort to support non-comment entity names. I've simplified the binding considerably, so it might be worth taking another look.